bench/box2d/optimized/distance.ll

```diff
@@ -1727,21 +1727,22 @@ b2ComputeSimplexClosestPoint.exit:                ; preds = %b2SolveSimplex2.exi
 
 209:                                              ; preds = %.critedge
   switch i32 %205, label %b2ComputeSimplexWitnessPoints.exit [
-    i32 2, label %213
+    i32 3, label %226
     i32 1, label %210
+    i32 2, label %213
   ]
```

Comment:
`correlated-propagation` is sensitive to the difference in PHI incoming value order, causing such regression.

original: 
```LLVM
%212 = phi i32 [ 1, %b2SolveSimplex2.exit.thread.thread ], [ 2, %203 ], [ %202, %b2SolveSimplex2.exit ]
```

new:
```LLVM
%212 = phi i32 [ 2, %203 ], [ %202, %b2SolveSimplex2.exit ], [ 1, %b2SolveSimplex2.exit.thread.thread ]
```